docs/docs/frontend-protocol.md

```diff
@@ -252,6 +252,34 @@ increase_number
 decrease_number
 ```
 
+#### Recording
+
+These methods allow manipulation and playback of event recordings.
+
+- If there is no currently active recording, start recording events under the provided name.
+- If there is no provided name, the current recording is saved.
```

Comment:
What if there is no current recording? Or is there an implicit default?

Reply:
If there is no current recording and no name is provided, nothing happens.